### ISSUE TITLE
我在win10上部署lyrebird，出现如下报错；能够帮我解决吗？或者能够我一份Windows的部署文档吗？

### ISSUE THREAD

**Issue by boziyoung** [Meituan-Dianping/lyrebird#427]

**Comment by boziyoung** [comment]
我的电脑环境：
win10专业版
anaconda3的python3.62 版本

报错是：
ERROR: Cannot uninstall 'ruamel-yaml'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.

我手动删除后者使用 uninstall 命令皆不奏效

**Comment by boziyoung** [comment]
C:\WINDOWS\system32>Lyrebird
Traceback (most recent call last):
  File "d:\userspython\boziyoung\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "d:\userspython\boziyoung\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\Userspython\boziyoung\Anaconda3\Scripts\lyrebird.exe\__main__.py", line 7, in <module>
  File "d:\userspython\boziyoung\anaconda3\lib\site-packages\lyrebird\manager.py", line 98, in main
    application._cm = ConfigManager()
  File "d:\userspython\boziyoung\anaconda3\lib\site-packages\lyrebird\config.py", line 37, in __init__
    self.read_config()
  File "d:\userspython\boziyoung\anaconda3\lib\site-packages\lyrebird\config.py", line 61, in read_config
    self.config_root), download_dir=str(self.ROOT/'downloads')))
  File "d:\userspython\boziyoung\anaconda3\lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "d:\userspython\boziyoung\anaconda3\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "d:\userspython\boziyoung\anaconda3\lib\json\decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Invalid \escape: line 6 column 21 (char 119)

**Comment by zhaoye** [comment]
现在在Windows上确实适配不好。

@boziyoung  可以尝试一下使用Docker镜像来启动。

1.首先安装 Docker Desktop
https://www.docker.com/products/docker-desktop

2.然后执行下面命令
```shell
docker run --rm -it -p 9090:9090 -p 4272:4272 -v d:\ userspython\.lyrebird:/root/.lyrebird -v d:\ userspython\.mitmproxy:/root/.mitmproxy overbridge/lyrebird
```
3.启动成功会在终端看到：
![image](https://user-images.githubusercontent.com/4516926/102034456-febe4c00-3df8-11eb-9968-ae92888a4d31.png)

4.这时在本地浏览器打开 http://localhost:9090 即可看到UI了

**Comment by boziyoung** [comment]
thank you

**Comment by woods-dev** [comment]
这是一个非常低级的错误。解决方法：修改配置文件，将 路径中的 \ 改成  \\

**Comment by zhaoye** [comment]
Fixed #435
